eng/code-quality-reports/src/main/resources/revapi/clientcore-revapi.json

```diff
@@ -0,0 +1,91 @@
+[
+  {
+    "extension": "revapi.java",
+    "configuration": {
+      "missing-classes": {
+        "behavior": "ignore",
+        "ignoreMissingAnnotations": false
+      },
+      "matchOverloads": true
+    }
+  },
+  {
+    "extension": "revapi.versions",
+    "configuration": {
+      "enabled": true,
+      "semantic0": false,
+      "versionIncreaseAllows": {
+        "major": {
+          "severity": "BREAKING"
+        },
+        "minor": {
+          "severity": "NON_BREAKING"
+        },
+        "patch": {
+          "severity": "EQUIVALENT"
+        }
+      },
+      "onAllowed": {
+        "remove": true,
+        "attachments": {
+          "breaksVersioningRules": "false"
+        }
+      },
+      "onDisallowed": {
+        "criticality": "error",
+        "attachments": {
+          "breaksVersioningRules": "true"
+        }
+      },
+      "passThroughDifferences": [
+        "java.class.nonPublicPartOfAPI"
+      ]
+    }
+  },
+  {
+    "extension": "allowed-external-apis",
+    "configuration": {
+      "enabled": true,
+      "allowedPrefixes": {
+        "io.": [
+          "clientcore.core"
```

Comment:
Should we everything with prefix `io.clientcore` instead of just `io.clientcore.core`?

Reply:
nah, let's keep the list closed initially.  so we can intentionally choose to widen the allow-list as new io.clientcore.* dependencies are added